travis-build.sh

```diff
@@ -5,12 +5,15 @@ ORG=${ORG:-hsldevcom}
 DOCKER_TAG=${TRAVIS_BUILD_NUMBER:-latest}
 DOCKER_IMAGE=$ORG/jore-map-ui:${DOCKER_TAG}
 DOCKER_IMAGE_LATEST=$ORG/jore-map-ui:latest
+DOCKER_IMAGE_DEV=$ORG/jore-map-ui:dev
```

Comment:
What is the difference between :dev and :latest?

Reply:
We usually have the practise of tagging images based on where they are used. :Latest is just a tag for the latest image.